js/widgets/toolbar.js

```diff
@@ -47,7 +48,7 @@ return $.widget( "mobile.toolbar", {
 
 	_create: function() {
 		var leftbutton, rightbutton,
-			role = this.element.is( ":jqmData(role='header')" ) ? "header" : "footer",
+			role =  this.element.is( ":jqmData(role='header')" ) ? "header" : "footer",
```

Comment:
@cgack @jaspermdegroot and i have been discussing this on IRC and i think this needs to work like this...
 * Toolbar initSelector will be moved to backcompat so it only use default of `data-role="toolbar"`.
 * Toolbar will get a new option `data-type` with options `header`, `footer` and `toolbar` default will be `toolbar`
  * The aria role will be set based on this option `header = banner`, `footer = contentinfo`, `toolbar = toolbar`
 * Toolbar will get a new option `data-aria-role` this will default to `null` which means use the mapping above. If a string use the string instead.